### ISSUE TITLE
The remote-control notification is not themed

### ISSUE THREAD

**Issue by CutterXYZ** [jellyfin/jellyfin-android#329]
Using Jellyfin for Android v0.9.9 and server 10.5.5 (but this bug already existed on 10.5.4).

I'm using my android device to control playback on my desktop. The notification appearing on my phone was usually themed to the content playing. Since a few days this notification is always white, despite the playing content being identified and populated with metadata.

See attached screenshot.

**Comment by rexbron** [comment]
Can you update to 1.0.0 and see if this is still an issue?

**Comment by CutterXYZ** [comment]
Yes, it still is.

**Comment by CutterXYZ** [comment]
Caddy is configured to encrypt everything and I was connecting to my server without specifying the https:// prefix. If I connect to the server with the https:// prefix this bug doesn't happen anymore.

**Comment by nielsvanvelzen** [comment]
Do you have a HTTP -> HTTPS redirect in Caddy? If that's the case the problem is probably not following redirects when resolving the image.

**Comment by CutterXYZ** [comment]
Yes Caddy responds to all requests with status code 308 that the app doesn't seem to follow.
